api/src/main/java/com/github/retrooper/packetevents/protocol/particle/data/ParticleVibrationData.java

```diff
@@ -174,6 +198,10 @@ public static void write(PacketWrapper<?> wrapper, ParticleVibrationData data) {
             wrapper.writeBlockPosition(data.getBlockPosition().get());
         } else if (data.getType() == PositionType.ENTITY) {
             wrapper.writeVarInt(data.getEntityId().get());
+            if (wrapper.getServerVersion().isNewerThanOrEquals(ServerVersion.V_1_19)
+                    && data.getEntityEyeHeight().isPresent()) {
+                wrapper.writeFloat(data.getEntityEyeHeight().get());
+            }
```

Comment:
IMO this should write a default value, even if no entity eye height has been specified. I would prefer having some strange values, instead of having an exception kick players from the server.